integration/simulation/network/obscuro_node_utils.go

```diff
@@ -248,12 +249,19 @@ func waitTimeout(wg *sync.WaitGroup, timeout time.Duration) bool {
 }
 
 func isAddressAvailable(address string) bool {
-	ln, err := net.Listen("tcp", address)
-	if err != nil {
-		return false
+	// `net.Listen` requires us to strip the protocol, if it exists.
+	addressNoProtocol := address
+	splitAddress := strings.Split(address, "://")
+	if len(splitAddress) == 2 {
+		addressNoProtocol = splitAddress[1]
 	}
+
+	ln, err := net.Listen("tcp", addressNoProtocol)
```

Comment:
Does this server ever get closed ?

Reply:
Well, that's the purpose of this method. It's here to check that the server closes (see the surrounding method - it gives up after 10 secs, with an err msg).

Reply:
Ah, yeah I see. I didn't see how this was working.

That ignore of the close was throwing me out. Would be nice to log the `_ = ln.Close()` error if there is one.

Reply:
Cool, fixed.